azext_iot/digitaltwins/command_map.py

```diff
@@ -45,6 +45,30 @@ def load_digitaltwins_commands(self, _):
         cmd_group.wait_command("wait", "wait_instance")
         cmd_group.command("reset", "reset_instance", confirmation=True, is_preview=True)
 
+    with self.command_group(
```

Comment:
For near-term release lets comment out or disable `az dt data-history`

Reply:
We should think about a feature flag pattern in the future (using an env var(s) for configuration).